### DISCUSSION TITLE
"goodname" input

### DISCUSSION THREAD

**Discussion by beetgroove** [kginicis-ts/INICIS_Stdpay#103] [Q&A]
Is it possible to have multiple inputs like:

<input type="text" name="goodname" value="test product">
...
input type="text" name="goodname" value="test product n">

**Comment by ts-inicis** [comment]
안타깝게도 하나의 상품명으로 수신됩니다.

**Reply by beetgroove** [reply]
Thank you for your reply.
So how to register more than one product in the order?

**Reply by ts-inicis** [reply]
결제 요청 시 상품명과 구매 금액을 분산하여 처리되길 원하시는 것으로 판단됩니다. 
해당 문의 내용에 도움이 되는 장바구니 기능 관련 자료입니다. 링크를 확인해 주세요. 

[웹표준 장바구니](https://github.com/kginicis-ts/INICIS_Stdpay/blob/main/%EC%9B%B9%ED%91%9C%EC%A4%80%EA%B2%B0%EC%A0%9C_%EC%9E%A5%EB%B0%94%EA%B5%AC%EB%8B%88_%EC%98%B5%EC%85%98%EA%B0%80%EC%9D%B4%EB%93%9C_V1.0.pdf)
